src/views/Gallery.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <ion-page>
+    <team-filter-header  @team-selected="searchValue = $event"/>
+    <ion-searchbar
+        show-cancel-button="focus"
+        :placeholder="$t('search')"
+        inputmode="search"
+        enterkeyhint="search"
+        :value="searchValue"
+        @ion-change="searchValue = $event.target.value"
+    />
+    <ion-content scroll-x="false" scroll-y="false">
+      <ion-spinner v-if="loading" />
+        <ion-grid>
+            <ion-row>
+                <ion-col size="4" v-for="(item, index) in filterPicture" :key="index"> 
+                    <img :src="item.file?.url" @click="viewPicture(item,index)"/> 
```

Comment:
maybe use `ion-thumbnail`-here to make sure these are nicely sized and square?!?